pkgs/by-name/av/av1an/package.nix

```diff
@@ -40,19 +43,24 @@ symlinkJoin {
   postBuild =
     let
       runtimePrograms =
-        [ (ffmpeg.override { inherit withVmaf; }) ]
+        [
+          vapoursynth
+          (ffmpeg.override { inherit withVmaf; })
+        ]
         ++ lib.optional withAom libaom
         ++ lib.optional withSvtav1 svt-av1
         ++ lib.optional withRav1e rav1e
         ++ lib.optional withVpx libvpx
         ++ lib.optional withX264 x264
         ++ lib.optional withX265 x265
-        ++ lib.optional withVmaf libvmaf;
+        ++ lib.optional withVmaf libvmaf
+        ++ lib.optional withMkvtoolnix mkvtoolnix-cli;
     in
     ''
       wrapProgram $out/bin/av1an \
+        --prefix LD_LIBRARY_PATH : ${vapoursynth}/lib \
```

Comment:
Also, `LD_LIBRARY_PATH` is highly discouraged. It would be helpful to put examples of the issue in the PR description, so that alternatives can be explored if needed (and reviewers can verify if the PR fixes the issue).

Reply:
Vapoursynth plugins (only tested lsmash, Av1an's recommended chunking method) don't work without LD_LIBRARY_PATH. Do you want me to include my lsmash derivation here for reference?

Here is an example: `av1an -e x264 -i sample.mkv -o /tmp/sample_enc.mkv -m lsmash`

Reply:
> Do you want me to include my lsmash derivation here for reference?

Sure, that'd be helpful.

Reply:
Hm I couldn't find a way around the LD_LIBRARY_PATH, patchelf didn't seem to help...

Reply:
Why is LD_LIBRARY_PATH discouraged?